dash/dash-renderer/src/actions/dependencies.js

```diff
@@ -238,8 +234,22 @@ function validateDependencies(parsedDependencies, dispatchError) {
             });
         });
 
-        findDuplicateOutputs(outputs, head, dispatchError, outStrs, outObjs);
-        findMismatchedWildcards(outputs, inputs, state, head, dispatchError);
+        if (hasOutputs) {
```

Comment:
re: @syl-oh 's comments about the boolean named `no_update` in the Python code, maybe `has_outputs` would be a good name for that boolean.

Reply:
indeed :)